packages/docs/src/scripts/components/Footer.jsx

```diff
@@ -32,63 +32,102 @@ function renderLinks(links) {
 
   return urls.map(function(url) {
     return (
-      <li key={url}>
+      <dd key={url} className="ds-u-font-size--small ds-u-margin-left--0">
         <a href={url} target="_blank">
           {links[url]}
         </a>
-      </li>
+      </dd>
     );
   });
 }
 
 const Footer = () => {
   return (
-    <footer className="ds-u-fill--primary-alt-lightest ds-u-padding-y--6 ds-u-md-padding-bottom--3">
-      <div className="ds-u-padding-x--3 ds-u-margin-x--0">
+    <footer className="ds-u-fill--gray-lightest">
+      <section className="ds-u-padding-y--4 ds-l-container">
+        <h2 className="ds-h4">Help make the CMS Design System better</h2>
+        <p className="ds-u-measure--wide">
+          We use Github issues to keep track of new component submissions, bugs,
+          design feedback, and any other suggestions you may have.
+        </p>
         <div className="ds-l-row">
-          <article className="ds-l-col--12 ds-l-lg-col--4 ds-u-margin-bottom--4">
-            <h2 className="ds-h4">Find an issue or have a feature request?</h2>
+          <div className="ds-l-col--12 ds-l-sm-col--4 ds-l-lg-col--4 ds-u-margin-bottom--2">
             <a
-              className="ds-c-button ds-c-button--primary"
-              href={githubUrl('issues')}
+              className="ds-c-button"
+              href={githubUrl(
+                'issues/new?assignees=&labels=Bug&template=bug_report.md&title='
+              )}
               target="_blank"
             >
-              Ask questions on GitHub
+              Report a bug
             </a>
-            <p>
-              <a href="https://forms.cms.gov/cms-wds-design-system-contact-form/responses/new">
-                Contact the CMSDS team
-              </a>
-            </p>
-            <p className="ds-text ds-u-color--primary-darkest ds-u-font-size--small ds-u-margin-top--3 ds-u-measure--base">
-              A federal government website managed by the Centers for Medicare &
-              Medicaid Services 7500 Security Boulevard, Baltimore, MD 21124
-            </p>
-          </article>
-          <article className="ds-l-col--12 ds-l-sm-col--6 ds-l-lg-col--4 ds-u-margin-bottom--4">
-            <h2 className="ds-h4">CMS &amp; HHS Websites</h2>
-            <ul className="ds-c-list ds-c-list--bare ds-u-font-size--small">
+          </div>
+          <div className="ds-l-col--12 ds-l-sm-col--4 ds-l-lg-col--4 ds-u-margin-bottom--2">
+            <a
+              className="ds-c-button"
+              href={githubUrl(
+                'issues/new?assignees=&labels=&template=general-issue.md&title='
+              )}
+              target="_blank"
+            >
+              Request a change
+            </a>
+          </div>
+          <div className="ds-l-col--12 ds-l-sm-col--4 ds-l-lg-col--4">
```

Comment:
The other 2 buttons both have `ds-u-margin-bottom--2`. Should this one also have the class, or can we remove the margin for all the buttons?